backend/src/middlewares/check-params-is-valid/check-params-is-valid.middleware.ts

```diff
@@ -0,0 +1,28 @@
+import { RequestHandler } from 'express';
+import { HttpError } from '~/exceptions/exceptions';
+import { ErrorMessage, HttpCode, RouterParams } from '~/common/enums/enums';
+import { MAX_POSSIBLE_ID } from '~/common/constants/constants';
+
+const checkParamsIsValid = (...params: RouterParams[]): RequestHandler => {
+  const handler: RequestHandler = (req, _res, next) => {
+
+    params.forEach((param) => {
+      const checkParam = req.params[param];
+
+      if (checkParam) {
+        if (isNaN(Number(checkParam)) || Number(checkParam) > MAX_POSSIBLE_ID) {
```

Comment:
```suggestion
      if (checkParam && isNaN(Number(checkParam)) || Number(checkParam) > MAX_POSSIBLE_ID) {
```
and add variable for it